{"variants":[{"paths":["\/documentation\/idverification365id\/idverificationerror\/servererror"],"traits":[{"interfaceLanguage":"swift"}]}],"sections":[],"metadata":{"role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"serverError","kind":"identifier"}],"symbolKind":"case","title":"IdVerificationError.serverError","roleHeading":"Case","modules":[{"name":"IdVerification365id"}],"externalID":"c:@M@IdVerification365id@E@IdVerificationError@IdVerificationErrorServerError"},"kind":"symbol","schemaVersion":{"major":0,"minor":3,"patch":0},"hierarchy":{"paths":[["doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id","doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError"]]},"abstract":[{"type":"text","text":"Happens when there is an error thrown by the 365id server"}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError\/serverError"},"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"platforms":["iOS"],"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"serverError","kind":"identifier"}]}]}],"references":{"doc://com.365id.sdk.idverification/documentation/IdVerification365id/IdVerificationError/serverError":{"title":"IdVerificationError.serverError","url":"\/documentation\/idverification365id\/idverificationerror\/servererror","type":"topic","kind":"symbol","abstract":[{"type":"text","text":"Happens when there is an error thrown by the 365id server"}],"role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"serverError"}],"identifier":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError\/serverError"},"doc://com.365id.sdk.idverification/documentation/IdVerification365id":{"url":"\/documentation\/idverification365id","role":"collection","title":"IdVerification365id","type":"topic","kind":"symbol","abstract":[],"identifier":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id"},"doc://com.365id.sdk.idverification/documentation/IdVerification365id/IdVerificationError":{"role":"symbol","url":"\/documentation\/idverification365id\/idverificationerror","type":"topic","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"kind":"identifier","text":"IdVerificationError"}],"navigatorTitle":[{"kind":"identifier","text":"IdVerificationError"}],"title":"IdVerificationError","kind":"symbol","identifier":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError","abstract":[{"type":"text","text":"Id Verification Errors used in the "},{"code":"onError(error:)","type":"codeVoice"},{"text":" method in the ","type":"text"},{"type":"codeVoice","code":"IdVerificationEventDelegate"}]}}}